lib/index.js

```diff
@@ -24,6 +24,10 @@
     if (diff.length)
       throw new OptionError('Missing required options: ' + diff.join());
 
+    //Make url an array for multiple webhooks
+    if(typeof opts.url === 'string')
```

Comment:
@iBicha Isn't it better if we can think of making the option name urls? The name url makes it feel like only one value is supported. But what could be the cleanest way without breaking support for url option too :)